VladosMeetupPlatform/VladosMeetupPlatform.API/VladosMeetupPlatform.API.csproj

```diff
@@ -4,13 +4,17 @@
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <GenerateDocumentationFile>True</GenerateDocumentationFile>
+    <ProduceReferenceAssembly>True</ProduceReferenceAssembly>
```

Comment:
I'm also not sure, why would You want to use this option: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-options/output#producereferenceassembly. I think You don't need it at all.